src/docs/asciidoc/getting_started.adoc

```diff
@@ -16,14 +16,33 @@ You can use the Hazelcast
 link:https://github.com/hazelcast/hazelcast-command-line[Command Line Interface (CLI)^]
 to install and start Hazelcast IMDG.
 
-Assuming you have the link:https://brew.sh/[Homebrew^] package manager, run the following commands:
+Run the following commands to install Hazelcast CLI:
 
-[source,bash,subs="attributes+"]
+[source,bash,indent=0,subs="verbatim,attributes",role="secondary"]
+.Homebrew
 ----
 $ brew tap hazelcast/hz
 $ brew install hazelcast
 ----
 
+[source,bash,indent=0,subs="verbatim,attributes",role="secondary"]
+.Debian
+----
+$ wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=hazelcast | sudo apt-key add -
+$ echo "deb http://dl.bintray.com/hazelcast/deb stable main" | sudo tee -a /etc/apt/sources.list
+$ sudo apt update && apt install hazelcast
+----
+
+[source,bash,indent=0,subs="verbatim,attributes",role="secondary"]
+.RPM
+----
+$ wget https://bintray.com/hazelcast/rpm/rpm -O bintray-hazelcast-rpm.repo
+$ sudo mv bintray-hazelcast-rpm.repo /etc/yum.repos.d/
+$ sudo yum install hazelcast
+----
+
+After the installation of Hazelcast CLI, the package managers will display the installation directory and usage information.
```

Comment:
Could we add a post-installation information example for one of the ways here?

Something similar to:

" ..... and usage information. The following is an example if you have used brew:

???
???
"